libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -76,9 +76,15 @@ AP_BattMonitor::init()
                 _num_instances++;
                 break;
             case AP_BattMonitor_Params::BattMonitor_TYPE_MAXELL:
+#if CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_PX4_V3
+                drivers[instance] = new AP_BattMonitor_SMBus_Maxell(*this, state[instance], _params[instance],
```

Comment:
something is wrong, as there isn't any difference with the previous code

Reply:
@khancyr san. Comment thanks.
I got information on Maxell 's battery at Pixhawk 1.
I could not get Maxell 's battery information with Pixhawk 2.
I do not know, SOLO battery and Maxell batteries are different from the internal bus and the external bus.
I think that if it connects to the same I2C connector, it will be the same bus.